frontend/src/app/modules/availability/new-slot/schedule-definition/schedule-definition.component.sass

```diff
@@ -49,3 +49,142 @@ select
+    border-right: 5px solid transparent
+    border-top: 5px solid
+    margin: 0 4px
+    color: rgba(0, 0, 0, 0.54)
+
+:host ::ng-deep .ng-dropdown-panel
+    background: #fff
+    left: 0
+
+:host ::ng-deep .ng-dropdown-panel.ng-select-bottom
+    top: calc(100% - 1.25em)
+    box-shadow: 0 5px 5px -3px rgba(0, 0, 0, 0.2), 0 8px 10px 1px rgba(0, 0, 0, 0.14), 0 3px 14px 2px rgba(0, 0, 0, 0.12)
+
+:host ::ng-deep .ng-dropdown-panel.ng-select-top
+    bottom: calc(100% - .84375em)
+    box-shadow: 0 -5px 5px -3px rgba(0, 0, 0, 0.2), 0 -8px 10px 1px rgba(0, 0, 0, 0.14), 0 -3px 14px 2px rgba(0, 0, 0, 0.12)
+
+:host ::ng-deep .ng-dropdown-panel.multiple .ng-option.selected
+    background: #fff
+
+:host ::ng-deep .ng-dropdown-panel.multiple .ng-option.marked
+    background: rgba(0, 0, 0, 0.04)
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-header
+    border-bottom: 1px solid rgba(0, 0, 0, 0.12)
+    padding: 0 16px
+    line-height: 3em
+    min-height: 3em
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-footer
+    border-top: 1px solid rgba(0, 0, 0, 0.12)
+    padding: 0 16px
+    line-height: 3em
+    min-height: 3em
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-optgroup
+    user-select: none
+    cursor: pointer
+    line-height: 3em
+    height: 3em
+    padding: 0 16px
+    color: rgba(0, 0, 0, 0.54)
+    font-weight: 500
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-optgroup.ng-option-marked
+    background: rgba(0, 0, 0, 0.04)
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-optgroup.ng-option-disabled
+    cursor: default
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-optgroup.ng-option-selected
+    background: rgba(0, 0, 0, 0.12)
+    color: #3f51b5
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-option
+    line-height: 3em
+    min-height: 3em
+    white-space: nowrap
+    overflow: hidden
+    text-overflow: ellipsis
+    padding: 0 16px
+    text-decoration: none
+    position: relative
+    color: rgba(0, 0, 0, 0.87)
+    text-align: left
+
+:host ::ng-deep [dir="rtl"] .ng-dropdown-panel .ng-dropdown-panel-items .ng-option
+    text-align: right
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-option.ng-option-marked
+    background: rgba(0, 0, 0, 0.04)
+    color: rgba(0, 0, 0, 0.87)
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-option.ng-option-disabled
+    color: rgba(0, 0, 0, 0.38)
+
+:host ::ng-deep [dir="rtl"] .ng-dropdown-panel .ng-dropdown-panel-items .ng-option.ng-option-child
+    padding-right: 32px
+    padding-left: 0
+
+:host ::ng-deep .ng-dropdown-panel .ng-dropdown-panel-items .ng-option .ng-tag-label
+    padding-right: 5px
+    font-size: 80%
+    font-weight: 400
+    color: rgba(0, 0, 0, 0.38)
+
+:host ::ng-deep [dir="rtl"] .ng-dropdown-panel .ng-dropdown-panel-items .ng-option .ng-tag-label
+    padding-left: 5px
+    padding-right: 0
```

Comment:
why do we need so much styles rewriting?

Reply:
this is style for time zone combo box (it's not a fully style that was imported previously). We need this styles for correct view of our combobox